src/build.py

```diff
@@ -853,27 +853,30 @@ def WriteEmscriptenConfig(infile, outfile):
                         EMSCRIPTEN_CONFIG_ASMJS)
   WriteEmscriptenConfig(os.path.join(SCRIPT_DIR, 'emscripten_config_vanilla'),
                         EMSCRIPTEN_CONFIG_WASM)
-  try:
-    # Build a C++ file with each active emscripten config. This causes system
-    # libs to be built and cached (so we don't have that happen when building
-    # tests in parallel). Do it with full debug output.
-    # This depends on binaryen already being built and installed into the
-    # archive/install dir.
-    os.environ['EMCC_DEBUG'] = '2'
-    configs = [EMSCRIPTEN_CONFIG_WASM] + (
-        [EMSCRIPTEN_CONFIG_ASMJS] if use_asm else [])
-    for config in configs:
+
+  configs = [EMSCRIPTEN_CONFIG_WASM] + (
+      [EMSCRIPTEN_CONFIG_ASMJS] if use_asm else [])
```

Comment:
I find this needlessly cryptic. How about just:

```
configs = [EMSCRIPTEN_CONFIG_WASM]
if use_asm:
    configs.append(EMSCRIPTEN_CONFIG_ASMJS)
```

I know this change has already merged, but anyway...